packages/patterns/src/components/create-pattern-modal.js

```diff
@@ -131,6 +131,7 @@ export function CreatePatternModalContents( {
 					categoryMap={ categoryMap }
 				/>
 				<ToggleControl
+					__nextHasNoMarginBottom
```

Comment:
## Testing instructions

In the Site Editor, go to Patterns ▸ Add New Pattern ▸ Add New Pattern.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/016e8827-8fa5-4e35-b451-70c15bb0acf2" alt="Add New Pattern modal, before" width="370">|<img src="https://github.com/user-attachments/assets/02b39761-fbab-4de8-9ce0-b25904cd92db" alt="Add New Pattern modal, after" width="373">|